WasteManagement/WasteContainer/doc/spec.md

```diff
@@ -2,10 +2,12 @@
 
 ## Description
 
-A waste container.
+A waste container. 
```

Comment:
Do you want to remove this extra blank space at the end or the line or is it fine as it is? :)